backend/proxy/secure_socks_proxy.go

```diff
@@ -51,7 +51,7 @@ var (
 		Namespace: "grafana",
 		Name:      "secure_socks_requests_duration",
 		Help:      "Duration of requests to the secure socks proxy",
-	}, []string{"code"})
+	}, []string{"code", "datasource", "datasourceType"})
```

Comment:
In Grafana we use snake case for label names: https://github.com/grafana/grafana/blob/98e3a01aff3dba6fe1e6f544eb95e9f1b523a11a/pkg/infra/httpclient/httpclientprovider/datasource_metrics_middleware.go#L48

This should be:

```suggestion
	}, []string{"code", "datasource", "datasource_type"})
```